pycroft/lib/user.py

```diff
@@ -93,6 +93,12 @@ def setup_ipv4_networking(host):
                     subnet=subnet)
         session.session.add(new_ip)
 
+@with_transaction
+def reset_password(myUser):
+    plain_password = user_helper.generate_password(12)
+    myUser.password = plain_password
+    return plain_password
```

Comment:
This is missing the second most important feature: Adding a log entry saying the password has been resetted (and thus the function is missing something like a `processor` parameter)